README.md

```diff
@@ -10,8 +10,21 @@ This repository contains TypeScript implementation of noise protocol, an encrypt
 
 ## Usage
 
-TBD
+TBD (package not yet published)
```

Comment:
- you could still add code example how to instantiate noise and how to pass it to libp2p config
- add badges (check secio for badges)
- add short description